node/streaming_out_response.js

```diff
@@ -86,10 +86,12 @@ StreamingOutResponse.prototype.setOk = function setOk(ok) {
         self.errorEvent.emit(self, errors.ResponseAlreadyStarted({
             state: self.state
         }));
+        return false;
```

Comment:
Regardless, we should return here indeed. Definitely should not change an already sent response.